edgelet/iotedge/src/config/super_config.rs

```diff
@@ -73,6 +77,26 @@ pub enum EdgeCa {
     },
 }
 
+#[derive(Debug, serde_derive::Deserialize, serde_derive::Serialize)]
+#[serde(tag = "method")]
+pub enum EdgeCaIssuanceMethod {
```

Comment:
Why a custom type instead of reusing `common_config::super_config::CertIssuanceMethod` ?

Reply:
Using `common_config::super_config::CertIssuanceMethod` also brings in the `SelfSigned` and `LocalCa` methods. Do we want those? I suppose `LocalCa` is fine, but `SelfSigned` is already equivalent to Quickstart here.

Reply:
Ok, fixed.

Reply:
(We should try to remove `SelfSigned` at some point, I think. It was added specifically for quickstart Edge CA but it was always kind of a hack, and when quickstart Edge CA was actually implemented it doesn't use it anyway.)